io.github.swordpuffin.hunt.json

```diff
@@ -0,0 +1,39 @@
+{
+    "id" : "io.github.swordpuffin.hunt",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "hunt",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "hunt",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/SwordPuffin/Hunt.git",
+                    "tag" : "v1.0.1"
```

Comment:
screenshots need to be direct links, not links to HTML pages https://github.com/SwordPuffin/Hunt/blob/fadf41e33832cb47bac84e48b79e73af08574a7b/data/io.github.swordpuffin.hunt.metainfo.xml.in#L24-L27

https://raw.githubusercontent.com/SwordPuffin/Hunt/refs/tags/v1.0.1/data/screenshots/Screenshot1.png
https://raw.githubusercontent.com/SwordPuffin/Hunt/refs/tags/v1.0.1/data/screenshots/Screenshot2.png